File: python/module1.py
import math

class Module1:
    def getFuelRequired(self, mass ):
        return math.floor(mass / 3) - 2
    
    def getRecursiveFuel(self, mass):
        return self.recursiveFuel(mass, 0)

    def recursiveFuel(self, mass, fuel):
        tempMass = self.getFuelRequired(mass);

        if (tempMass <= 0):
            return fuel
        
        fuel += tempMass;
        
        return self.recursiveFuel(tempMass, fuel)

File: go/module1.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"math"
	"strconv"
	"strings"
)

func main() {
	reader := bufio.NewReader(os.Stdin)

	fmt.Print("What is the mass? ")
	input, _ := reader.ReadString('\n')
	input = strings.Replace(input, "\n", "", -1)

	mass, _ := strconv.Atoi(input)

	fuelRequired := getFuelRequired(mass);
	fmt.Println("Fuel Required:", fuelRequired)
}

func getFuelRequired(mass int)  int {
	return int(math.Floor(float64(mass / 3))) - 2
}